### DISCUSSION TITLE
What's the general process for removing a feature toggle?

### DISCUSSION THREAD

**Discussion by gravitylow** [grafana/grafana#67446] [General]
Support for CloudWatch cross-account observability was [added](https://github.com/grafana/grafana/commit/254577ba568d8010dcc96446e788866d618c96c8) in November 2022 and has been gated behind the `cloudWatchCrossAccountQuerying` feature toggle since then.

Unfortunately my team uses [Amazon Managed Grafana](https://aws.amazon.com/grafana/), which does not currently support modifying feature toggles (I have put a [request](https://github.com/aws/amazon-managed-grafana-roadmap/issues/44) on their roadmap), so we are unable to use this functionality currently and would really like to.

I'm curious about the general process of deprecating a feature toggle. At what point can we un-gate this functionality? Is cross-account observability anywhere close to GA?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
